pkgs/development/python-modules/dm-sonnet/default.nix

```diff
@@ -1,6 +1,6 @@
 { lib
 , fetchFromGitHub
-, bazel_0_26
+, bazel
```

Comment:
@timokau I've switched this from using the super old pinned version of bazel without updating the package since it's broken. Given that it's related to tensorflow I assume dm-sonnet would be fine using whichever version of bazel tensorflow is building with. Let me know if that assumption is wrong.

Reply:
Thanks for working on this. I haven't used `dm-sonnet` for a while. Since its already broken, there is probably little harm in changing this. You could leave a comment mentioning that the last known-good bazel version is 0.26.

Would you mind removing me from the maintainers list?

Reply:
Sure I can do that in a separate PR, you're the only maintainer though, should we delete the package? It's only used in `graph_nets` of which you're maintainer, should we remove that package too?

Reply:
Yeah we could just remove it, that would simplify the bazel change as well. I think it would be fine to do it in this PR (but a separate commit), but if you prefer a separate one I'd be glad to review it.

Reply:
https://github.com/NixOS/nixpkgs/pull/170073